packages/aws-cdk/lib/cli.ts

```diff
@@ -271,7 +271,8 @@ async function parseCommandLineArguments(args: string[]) {
     .command('init [TEMPLATE]', 'Create a new, empty CDK project from a template.', (yargs: Argv) => yargs
       .option('language', { type: 'string', alias: 'l', desc: 'The language to be used for the new project (default can be configured in ~/.cdk.json)', choices: initTemplateLanguages })
       .option('list', { type: 'boolean', desc: 'List the available templates' })
-      .option('generate-only', { type: 'boolean', default: false, desc: 'If true, only generates project files, without executing additional operations such as setting up a git repo, installing dependencies or compiling the project' }),
+      .option('generate-only', { type: 'boolean', default: false, desc: 'If true, only generates project files, without executing additional operations such as setting up a git repo, installing dependencies or compiling the project' })
+      .option('project-name', { type: 'string', desc: 'Name of the generated project (defaults to current directory name)', requiresArg: true }),
```

Comment:
Let me know if you would also like me to describe the new option in `aws-cdk/README.md`, or if this `--help` output is sufficient